docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -0,0 +1,173 @@
+---
+ms.date: 03/29/2019
+description: Create custom functions' JSON file automatically with JSDOC comments. 
+title: Generate JSON metadata for custom functions (preview)
+localization_priority: Priority
+---
+
+# Generate JSON metadata for custom functions (preview)
+
+When an Excel custom function is written in JavaScript or TypeScript, JSDoc tags are used to provide the extra information about the custom function.
+
+Add the `@customfunction` tag in the comments for a JavaScript or TypeScript function to mark it as a custom function.
+
+The function parameter types may be provided using the [@param](#param) tag in JavaScript, or from the [Function type](http://www.typescriptlang.org/docs/handbook/functions.html) in TypeScript. For more information, see the [@param](#param) tag and [Types](#Types) section.
+
+## Tags
+* [@cancelable](#cancelable)
+* [@customfunction](#customfunction) id name
+* [@helpurl](#helpurl) url
+* [@param](#param) _{type}_ name description
+* [@requiresAddress](#requiresAddress)
+* [@returns](#returns) _{type}_
+* [@streaming](#streaming)
+* [@volatile](#volatile)
+
+---
```

Comment:
The following sections read like an SDK. Can you soften the tone up a bit so it's not so rigid and maybe provide more examples.